test/Negative.test.js

```diff
@@ -1,15 +1,12 @@
-const math = require('mathjs');
-
-const flatten = require('../lib/util/flattenOperands');
 const print = require('../lib/util/print');
 
 const Negative = require('../lib/Negative');
 
 const TestUtil = require('./TestUtil');
 
 function testNegate(exprString, outputStr) {
-  const inputStr = Negative.negate(flatten(math.parse(exprString)));
-  TestUtil.testFunctionOutput(print, inputStr, outputStr);
+  const inputStr = Negative.negate(TestUtil.flattenAndParse(exprString));
```

Comment:
yay thanks for updating this everywhere - would you mind find+replacing `flattenAndParse` everywhere with `parseAndFlatten`? heh. just cause it happens in that order and therefore that'd be a bit clearer